CONTRIBUTING.md

```diff
@@ -161,65 +403,135 @@ make integration-test-container SSH_KEY_PATH=<path to gitops repo key> \
+  whoever created it to provide more information.
+  If they do not respond, or more help is needed,
+  you can then bring in one of the [core maintainers](COMMUNITY.md#core-team).
 
-- Make sure the tests are running successfully.
+- If you are struggling with something while working on your PR, or aren't quite
+  sure of your approach, you can open a [draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+  (prefix the title with `WIP: `) and explain what you are thinking.
+  You can tag in one of the core team, or drop the PR link into [slack](https://weave-community.slack.com/messages/eksctl/) and get
+  help from the community.
 
-### Submitting a pull request
+- We are happy to pair with contributors over a slack call to help them fine-tune their
+  implementation. You can ping us directly, or head to the [channel](https://weave-community.slack.com/messages/eksctl/)
+  to see if anyone in the community is up for being a buddy :smiley: .
 
-Push your changes to your fork and submit a pull request to the original repository. If your PR is a work in progress
-then make sure you prefix the title with `WIP: `. This lets everyone know that this is still being worked on. Once its
-ready remove the `WIP: ` title prefix and where possible squash your commits. Alternatively, you can use `Draft PR`
-feature of Github as mentioned [here](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+## PR submission guidelines
+
+Push your changes to the branch on your fork and submit a pull request to the original repository
+against the `master` branch.
+Where possible, please squash your commits to ensure a tidy and descriptive history.
 
 ```bash
-git push <username> <feature-name>
+git push <remote-name> <feature-name>
 ```
 
+If your PR is still a work in progress, please open a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+
 Our CircleCI integration will run the automated tests and give you feedback in the review section. We will review your
-changes and give you feedback as soon as possible.
+changes and give you feedback as soon as possible. We also encourage people to post
+links to their PRs in slack to get more eyes on the work.
+
+We recommend that you regularly rebase from master of the original repo to keep your
+branch up to date.
+
+PRs which adhere to our guidelines are more likely to be accepted
+(when opening the PR, please use the checklist in the template):
+
+1. **The description is thorough.** When writing your description, please be as detailed as possible: don't make people
+  guess what you did or simply link back to the issue (the issue explains the problem
+  you are trying to solve, not how you solved it.)
+  Guide your reviewers through your solution by highlighting
+  key changes and implementation choices. Try and pre-empt any obvious questions
+  they may have. Providing screenshots or snippets of output is very helpful to
```

Comment:
Snippets > screenshots, makes searching the git issue a lot easier (although I do enjoy a good sneak peak at peoples terminal profiles)